.github/workflows/smoke-test.yaml

```
@@ -0,0 +1,41 @@
+
+    env:
+      PROJECT_NAME: link
+      HEALTH_CHECK_TIMEOUT: 10
+      CHECK_INTERVAL: 5
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)